packages/fxa-auth-server/bin/key_server.js

```diff
@@ -31,19 +31,14 @@ const { Container } = require('typedi');
 const { StripeHelper } = require('../lib/payments/stripe');
 const { PlayBilling } = require('../lib/payments/iap/google-play');
 const { CurrencyHelper } = require('../lib/payments/currencies');
-const {
-  AuthLogger,
-  AuthFirestore,
-  AppConfig,
-  ProfileClient,
-} = require('../lib/types');
+const { AuthLogger, AuthFirestore, AppConfig } = require('../lib/types');
 const { setupFirestore } = require('../lib/firestore-db');
 const { AppleIAP } = require('../lib/payments/iap/apple-app-store/apple-iap');
 const { AccountEventsManager } = require('../lib/account-events');
 const { AccountDeleteManager } = require('../lib/account-delete');
 const { gleanMetrics } = require('../lib/metrics/glean');
 const Customs = require('../lib/customs');
-const Profile = require('../lib/profile/client');
+const { ProfileClient } = require('@fxa/profile/client');
```

Comment:
I forget - this type of import will work fine in all envs now, right?

Reply:
Yup. In the case of `fxa-auth-server` as long as the file is picked up by the build process, which runs `tsc-alias`, it should be good to go.